astronomer/providers/apache/livy/operators/livy.py

```diff
@@ -79,4 +91,5 @@ def execute_complete(self, context: Context, event: Dict[str, Any]) -> Any:
             self.task_id,
             event["response"],
         )
+        context["ti"].xcom_push(key="app_id", value=self.get_hook().get_batch(event["batch_id"])["appId"])
```

Comment:
I'm not sure, but I feel `self.get_hook()` may be none here wdyt? would it be possible to run DAG once?